data/templates/install-coreos.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 curl -O http://<%=server%>:<%=port%>/api/current/templates/pxe-cloud-config.yml
 sudo coreos-install -d <%=installDisk%> -c pxe-cloud-config.yml -b <%=repo%>
+wget --spider http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%>
```

Comment:
The task definition (https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/install-coreos.js#L18) still need be changed, otherwise the install-coreos will still finish after the config file is downloaded. 

Since both completionUri and notification API mechanism co-exist before we completely migrate to notification API. So at present, if you want to use the notification API, you need change the `completionUri` in task definition to something arbitrary (usually "renasar-ansible.pub") that prevents the completionUri from being work.

Add @lanchongyizu for awareness since he is the owner to migrate all install-os to use new notification API.